Sources/MongoSwift/BSON/AnyBSONValue.swift

```diff
@@ -5,21 +5,23 @@ import Foundation
 public struct AnyBSONValue: Codable, Equatable, Hashable {
     // TODO: conform all `BSONValue` types to `Hashable` (SWIFT-320).
     // swiftlint:disable:next legacy_hashing
```

Comment:
we need to remove this "disable" line, as SwiftLint now considers it superfluous.